modules/deploying-models-on-single-node-openshift-using-kserve-raw-deployment-mode.adoc

```diff
@@ -0,0 +1,256 @@
+:_module-type: PROCEDURE
+
+[id="deploying-models-on-single-node-openshift-using-kserve-raw-deployment-mode_{context}"]
+= Deploying models on single node openshift using kserve raw deployment mode
+
+[role='_abstract']
+You can deploy a machine learning model by using KServe raw deployment mode on single node OpenShift. With raw deployment mode, you can overcome the limitations of Knative, such as gaining the ability to mount multiple volumes.
+
+[IMPORTANT]
+====
+Deploying a machine learning model using KServe raw deployment mode on single node OpenShift is a Limited Availability feature. Limited Availability means that you can install and receive support for the feature only with specific approval from the Red Hat AI Business Unit. Without such approval, the feature is unsupported.
+====
+
+.Prerequisites
+* You have logged in to {productname-long}.
+* You have cluster administrator privileges for your OpenShift Container Platform cluster.
```

Comment:
{openshift-platform} instead of _OpenShift Container Platform_ cluster?

Reply:
Resolved to your suggestion.